src/Command/CommandBase.php

```diff
@@ -857,12 +858,12 @@ protected function doDetermineCloudApplication() {
     }
 
     // Try to guess based on local git url config.
-    if ($cloud_application = $this->inferCloudAppFromLocalGitConfig($acquia_cloud_client)) {
+    if ($prompt_user && $cloud_application = $this->inferCloudAppFromLocalGitConfig($acquia_cloud_client)) {
```

Comment:
Same as previous comment but just to reiterate, `$prompt_user` appears to always be TRUE and can be removed.